examples/research_projects/stack_llama_2/scripts/sft_llama2.py

```diff
@@ -5,6 +5,7 @@
 
 import torch
 from accelerate import Accelerator
+from accelerate.utils import is_xpu_available
```

Comment:
`is_xpu_available` has been only introduced in 0.20.0 - can you maybe create an utility method `def is_xpu_available()` in trl to 
1- check if accelerate version is greater than 0.20.0
2- if yes, call `accelerate.utils.is_xpu_available`
3- else, return `False`

That way users that have an old version of accelerate should be able to use TRL without any problem.

Reply:
Agree with the logic flow. I also added a smaller utility to support xpu even if accelerate version is before 0.20.0 which can work until the accelerate package gets upgraded .